client/client.go

```diff
@@ -423,13 +425,83 @@ func (c *Client) getLocalMeta() error {
 			c.loadTargets(targets.Targets)
 		}
 	}
+
+	if loadFailed {
+		// If any of the metadata failed to be verified, return the reason for that failure
+		// and fail fast before delegated targets
+		return retErr
+	}
+
+	// verifiedDelegatedTargets is a set of verified delegated targets
```

Comment:
can we run this code below line 457 so we remove the duped `if loadFailed` code?

Reply:
@asraa are we good to merge this? WDYT?

Reply:
Ah! What I didn't see was that it's possible loadFails below as well, so both are necessary (both before we try to verifyDelegatedTargets, and after, in case those manifests fail to load.

yes!